### DISCUSSION TITLE
Code base SLOC statistics support (via `tokei` or alternative tools)

### DISCUSSION THREAD

**Discussion by evan0greenup** [community/community#113250] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Currently, the language statistics for code of a repository is like below:

![Screenshot from 2024-03-19 17-31-45](https://github.com/community/community/assets/44539531/02c17aea-593c-43f7-95b3-4b9fa58a5bdd)

Users can only know how much portion each language occupy, but do not know their exact quantity.

It would be nice to also include a table of SLOC statics below this portion bar.

It can make users be able to estimate the scale of a project.

A wonderful tool to do this is `tokei` <https://github.com/XAMPPRocky/tokei>.

**Comment by xi** [comment]
I would love to have this. When looking for libraries, the size of the code base is a very relevant factor for me.
